kube-aws-updater

```diff
@@ -246,45 +237,138 @@ cycle_nodes() {
   local ic=0
   while [[ ${ic} -ne ${asg_count} ]]; do
     sleep 32
-    ic=$(retry aws ${aws_opts} --output=json autoscaling describe-auto-scaling-groups \
+    ic=$(retry aws "${aws_opts[@]}" --output=json autoscaling describe-auto-scaling-groups \
       --auto-scaling-group-names "${asg_name}" |\
-      jq -r -e '."AutoScalingGroups"[0]."Instances" | length')
+      jq -r -e '.AutoScalingGroups[0].Instances | length')
```

Comment:
I don't mind general format changes, but why this one?

Reply:
For consistency with the rest of `jq` queries and because I find it very odd and confusing that it's quoted when there's no need for quotes.